internal/gatewayapi/backendtrafficpolicy.go

```diff
@@ -309,6 +309,12 @@ func (t *Translator) translateBackendTrafficPolicyForRoute(
 		err, errs error
 	)
 
```

Comment:
can you rm all the changes in `internal` from this `api` PR, and add these in a subsequent PR